`kubernetes_cluster/node_pool`- `capacity_reservation_group_id` (#17395)
hashicorp/terraform-provider-azurerm: website/docs/r/kubernetes_cluster.html.markdown

* `vm_size` - (Required) The size of the Virtual Machine, such as `Standard_DS2_v2`. Changing this forces a new resource to be created.

> * `capacity_reservation_group_id` - (Optional) AKS will associate the specified agent pool with the Capacity Reservation Group. Changing this forces a new resource to be created.

Reviewer: presumably this should be:

```suggestion
* `capacity_reservation_group_id` - (Optional) Specifies the ID of the Capacity Reservation Group within which this AKS Cluster should be created. Changing this forces a new resource to be created.
```
